recipe/build.sh

```diff
@@ -37,34 +33,18 @@ case "$target_platform" in
     ;;
 esac
 
+# https://github.com/openssl/openssl/issues/16546
+if [ "${}" = "linux-"* ]; then
+  sed -i "s@127.0.0.1@$(hostname -I)@g" test/recipes/80-test_cmp_http.t test/recipes/80-test_cmp_http_data/Mock/test.cnf
```

Comment:
Thanks for helping out here @chrisburr!

I think this is going to need to apply to both linux & osx.

Reply:
`hostname -I` doesn't work on macOS, I was hoping to grep it from `ifconfig`

Reply:
Ah, apologies, will reinstate with the next push.